Content.Server/Atmos/Gas.cs

```diff
@@ -0,0 +1,10 @@
+namespace Content.Server.Atmos
+{
+    public enum Gas
```

Comment:
This is something that can be done later, but we'll probably want gas definitions to be loaded from yaml prototypes instead of being hardcoded enums. See [ReagentPrototype.cs](https://github.com/space-wizards/space-station-14/blob/master/Content.Shared/Chemistry/ReagentPrototype.cs) for an example of this. Reagents using that prototype are in `Resources/Prototypes/Reagents/`. You can use `IPrototypeManager` to access prototypes.